contracts/src/Teleporter/ReceiptQueue.sol

```diff
@@ -56,16 +64,11 @@ library ReceiptQueue {
     ) internal returns (TeleporterMessageReceipt[] memory result) {
         // Get the current outstanding receipts for the given chain ID.
         // If the queue contract doesn't exist, there are no outstanding receipts to send.
```

Comment:
Good call. I don't think the above comment applies any more either since technically the queue exists for every chain ID in the mapping (with the default value if it has never been set previously). I'll update it.